docs/docsite/rst/community/steering/steering_committee_membership.rst

```diff
@@ -65,7 +65,7 @@ Voluntarily leaving the Steering Committee
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 A Committee member can voluntarily leave the Committee.
-In this case, they notify the other members, create an issue in the `Community Topics <https://github.com/ansible-community/community-topics/issues>`_ repository announcing the resignation, and after that, they are no longer considered Committee members.
+In this case, the member notifies the other members, creates a `Community Topic <https://forum.ansible.com/tags/c/project/7/community-wg>`_ announcing their resignation, and after that, they are no longer considered a Committee member.
```

Comment:
```suggestion
In this case, the member notifies the other members, creates a `Community Topic <https://forum.ansible.com/tags/c/project/7/community-wg>`_ announcing their resignation, and after that, they are no longer considered a Committee member.
```
I suggest doing this via PM to SC also, see the justification in https://forum.ansible.com/t/voluntarily-leaving-the-steering-committee/3952/4

Reply:
Another option can be a section in Bullhorn called `Steering Committee updates` or something saying just a fact that someone joined/resigned with gratitude w/o reasons at least in case of stepping down.